File: Unmatched.UI.Angular/src/app/backend-api/models/hero-dto.ts
/* tslint:disable */
/* eslint-disable */
import { PlayStyleDto } from './play-style-dto';
import { SidekickDto } from './sidekick-dto';
import { TitleDto } from './title-dto';
export interface HeroDto {
  color?: null | string;
  deckSize?: number;
  hp?: number;
  id?: string;
  imageUrl?: null | string;
  isRanged?: boolean;
  meleeRangeImageUrl?: null | string;
  name?: null | string;
  playStyle?: PlayStyleDto;
  sidekicks?: null | Array<SidekickDto>;
  titles?: null | Array<TitleDto>;
}

File: Services/Catalog/Unmatched.CatalogService.Api/Dto/HeroDto.cs
namespace Unmatched.CatalogService.Api.Dto;

using System;


public class HeroDto
{
    public int DeckSize { get; set; }

    public int Hp { get; set; }

    public Guid Id { get; set; } 

    public bool IsRanged { get; set; }

    public string Name { get; set; }

    //public virtual ICollection<Sidekick> Sidekicks { get; set; }

    public string Color { get; set; }

    //public virtual ICollection<Title> Titles { get; set; }

    //public virtual PlayStyle? PlayStyle { get; set; }
}
